components/eam/src/chemistry/yaml/aero_model/modal_aero_bcscavcoef_init_input_ts_0.yaml

```diff
@@ -0,0 +1,10 @@
+# Data at [lat=  0.000000000000000E+000 , lon=  0.000000000000000E+000 , k=           0 , time step=         0 ] 
+mam4xx:
+  function: modal_aero_bcscavcoef_init
+input:
+  fixed:
+    dt:    0.00
+    dgnum_amode: [ 0.1100000000E-06, 0.2600000000E-07, 0.2000000000E-05, 0.5000000058E-07]
+    sigmag_amode: [ 0.1800000000E+01, 0.1600000000E+01, 0.1800000000E+01, 0.1600000024E+01]
+    specdens_amode: [ 0.1770000000E+04, 0.1797693135+309, 0.1797693135+309, 0.1000000000E+04]
```

Comment:
what is this 0.1797693135+309 in `specdens_amode`?

Reply:
looks like this is the mode/species that are not exist in the current model (it reads `specdens_amode(ll)` and `ll = lspectype_amode(1,imode)`, while `lspectype_amode` does not have number 2 and 3). I believe we can simply ignore it. @singhbalwinder do you have any comment on it?